src/app/dataset-block/metadata-block/metadata-block.component.html

```diff
@@ -28,8 +28,13 @@
     <div class="container-main row">
         <div class="col-md-6">
             <div class="container-content mb-4">
-                <app-block-header [block]="metadata.block" [datasetInfo]="datasetInfo" />
-                <app-event-details [block]="metadata.block" [datasetInfo]="datasetInfo" />
+                <app-block-header
+                    [block]="metadata.block"
```

Comment:
Since YAML view already includes the block header I think we should hide this section under `*ngIf="!yamlView"`